Readme.md

```diff
@@ -2,40 +2,44 @@
+Do you want to quickly try out this project and see it in action? Check it out, together with its accompanying [samples](Lombiq.NodeJs.Extensions.Samples) [projects](Lombiq.NodeJs.Extensions.Samples.NuGet), in our [Open-Source Orchard Core Extensions](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions) full Orchard Core solution. You will find our other useful Orchard Core-related open-source projects there, too.
 
 
 ## Installation and usage
 
-It's recommended to put this project into a folder named _Lombiq.NodeJs.Extensions_ under the _src/Utilities_ folder, but you can put it anywhere. Then add it as an `npm` package to your own project running:
-
-```
-npm install --save-dev ../../Utilities/Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions
-```
-This will be the path if your project resides in a child of the *src* folder. Adjust it to your needs.
-
-Afterwards, you can use any of the [available scripts](#available-scripts) in this package.
+This project can be consumed as a `git` submodule or as a `NuGet` package.
 
+### As a git submodule
+In the case of using it as a git submodule, it is recommended to put the [main](../Lombiq.NodeJs.Extensions) project into a folder named _Lombiq.NodeJs.Extensions_ under the _src/Utilities_ folder, but this is not mandatory.
 
-### Integrating with MSBuild
-
-If you want to integrate Stylelint into MSBuild builds, then you need to include Lombiq's [NPM MSBuild Targets](https://github.com/Lombiq/NPM-Targets), too. Make sure you have a _package.json_ file with the `dotnet-prebuild` and `dotnet-postclean` scripts as indicated in the repository's readme. In the affected projects, you need to import these files in the `.csproj` file:
+Then, add a project reference to _Lombiq.NodeJs.Extensions.csproj_ and add the following `Import` statements to your project file:
 
 ```xml
-<Import Project="..\..\Utilities\Lombiq.Npm.Targets\Lombiq.Npm.Targets.props" />
-<Import Project="..\..\Utilities\Lombiq.Npm.Targets\Lombiq.Npm.Targets.targets" />
+<Import Project="..\..\Utilities\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.props"/>
 <Import Project="..\..\Utilities\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.targets"/>
 ```
+In case you've placed the submodule in a different location, adjust the paths as necessary.
+
+### As a NuGet package
+
+When adding _Lombiq.NodeJs.Extensions_ as a NuGet package, no further steps are necessary.
 
-Then, warnings will be sent to the error list if the linter finds rule violations.
+### Integration with MSBuild
+
+_Lombiq.NodeJs.Extensions_ tightly integrates with the MSBuild build system and executes linting and compilation tasks transparently. In case of warnings or errors during the execution of those tasks, respective warnings and errors will be generated with MSBuild and brought to your attention.
```

Comment:
This is a technical identifier (as opposed to "Lombiq Node.js Extensions"), thus needs to be treated as code. Also in the other places.

```suggestion
`Lombiq.NodeJs.Extensions` tightly integrates with the MSBuild build system and executes linting and compilation tasks transparently. In case of warnings or errors during the execution of those tasks, respective warnings and errors will be generated with MSBuild and brought to your attention.
```